demo/src/screens/commonScreens/index.js

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
I recently did some work and reducing hierarchy of our folders 
let's put this example screen with the rest of the components

Reply:
Moved